### ISSUE TITLE
grunt is not working when i activated, Html5 mode.

### ISSUE THREAD

**Issue by akhilclarion** [ngbp/ngbp#399]
Hi, 

I have removed "#" from angular routing url. i mean $locationProvider.html5Mode(true);

After doing above changes, when i refreshed the page and it throws the error "URL is not found"

**Comment by SailingYYC** [comment]
In order to use HTML 5 Mode, you need to be hosting the site on a specially configured web server.  The server needs to be configured to internally redirect all requests for not found files to the appropriate HTML file containing your single page app.

I can't tell from your description above if this is how you have things configured or at what stage in the development / release cycle you're referring.  Have you researched the documentation on AngularJS's $locationProvider.html5Mode?

**Comment by joshdmiller** [comment]
I agree with @SailingYYC. Can you provide some more information? Which server are you using? From where is the error coming (browser or server)? What does the URL look like at that point?
